### ISSUE TITLE
no-unsafe-regex: Probably false positive

### ISSUE THREAD

**Issue by ilyub** [sindresorhus/eslint-plugin-unicorn#1790]
I get "no-unsafe-regex" warning for this code:
```ts
/(?:\/\w+){2}/u
```

but no warning for:
```ts
/\/\w+\/\w+/u
```

But these regular are identical. They should either both show warning or both don't show warning.

**Comment by fregante** [comment]
`no-unsafe-regex` uses `safe-regex` under the hood. We can't fix that here.

Follow https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2025 to see where to report the issue
